### ISSUE TITLE
Unchecking an overridable field is not detected as a change

### ISSUE THREAD

**Issue by simmonsr** [gantry/gantry5#333]
With our Font Families, I have Aldrich as the Promo Font in Home Outline, and Akronim as the Promo Font set up in Base in Home Outline.

If I unselect Aldrich in Home Outline, I'm not prompted to "Save Styles" in Home (shouldnt i be?). And even if i do "Save Styles" in Home, I have to "Save Styles" in Base also for it to revert back to Akronim.

**Comment by mahagr** [comment]
@simmonsr You closed the issue, was it invalid?

**Comment by simmonsr** [comment]
Djamil said it was related to https://github.com/gantry/gantry5/issues/326

**Comment by w00fz** [comment]
Actually just the font issue you were having was related to #326, this one is still a valid issue.

**Comment by simmonsr** [comment]
This issue is not present in RC3 though

**Comment by simmonsr** [comment]
Other than not being "prompted to save". I can open a new issue for that if you would like.

**Comment by mahagr** [comment]
Let's just rename this one.

**Comment by w00fz** [comment]
Was about to implement this but we kind of need the whole Reverting to Defaults implementation before we can work on this.

**Comment by mahagr** [comment]
@w00fz Did you already fix this one?

**Comment by w00fz** [comment]
Almost an year later... this is now implemented 😸
